libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -68,7 +68,7 @@
  * @see <a href="https://www.mapbox.com/android-docs/navigation/">Navigation documentation</a>
  * @since 0.1.0
  */
-public class MapboxNavigation implements ServiceConnection {
+public class MapboxNavigation implements ServiceConnection, MapboxNavigationInterface {
```

Comment:
Opinion: give interfaces the clean name and the implementation the extra. We'll maybe hide the implementation of the interface.

interface MapboxNavigation
class MapboxNavigationImpl

interface LocationProvider
class AndroidLocationProvider, MockLocationProvider